docs/libs/creating-lib.md

```diff
@@ -292,15 +292,15 @@ Use the following template:
 }
 ```
 
-- `tsconfig.spec.json` - add `"target": "es2020", "module": "es2020"` in `"compilerOptions"`:
+- `tsconfig.spec.json` - add `"target": "es2022", "module": "es2022"` in `"compilerOptions"`:
```

Comment:
[QUESTION]
(not related to this file, but to es2020 in general):

Should we also change this place to es2022? https://github.com/SAP/spartacus/blob/26c6c14939bb2512952fb95a9826558e05d7700c/tools/config/manage-dependencies.ts#L165

Reply:
good catch, thanks!